README.md

```diff
@@ -32,11 +44,16 @@ in your shell:
 $ eval $(./taskcluster signin --name taskcluster-web-server)
```

Comment:
Setting the credentials can also be done via `user-config.yml`.

Reply:
It's true, but I don't want to encourage that, since (a) it means writing down credentials somewhere that's easy to copy/paste by mistake and (b) the credentials are temporary, so you'll end up having to edit that file frequently.  The `taskcluster signin` option avoids both of those pitfalls.

Reply:
Fair enough.